crates/uv-build/src/lib.rs

```diff
@@ -723,7 +722,7 @@ impl SourceBuild {
     #[instrument(skip_all, fields(version_id = self.version_id))]
     pub async fn build_wheel(&self, wheel_dir: &Path) -> Result<String, Error> {
         // The build scripts run with the extracted root as cwd, so they need the absolute path.
-        let wheel_dir = fs::canonicalize(wheel_dir)?;
+        let wheel_dir = std::path::absolute(wheel_dir)?;
```

Comment:
I also changed this from canonicalize to absolute, since the error isn't very good if the directory doesn't exist.